### ISSUE TITLE
Example document

### ISSUE THREAD

**Issue by TotallyAProgrammer** [Bluemoondragon07/Wikipedia-Theme#15]
Can you please upload the document you used in the demonstration photo? I have the theme functional, but was curious to see how you went about formatting everything in the example given.

**Comment by vec715** [comment]
agree, I can't understand how to paste image in a wiki style on the right side of the screen

**Comment by Bluemoondragon07** [comment]
Sorry this is very late!. Here is the file content: 

```md

> Created by: **Ha'ani Whitlock**


>[!info] 
>![[article-example.png]]
>[Creator:: Ha'ani Whitlock]
>[Created:: December 16, 2022]
>[App:: Obsidian.md]
>[Supported Themes:: Light, Dark]
>[Description:: A simple, nostalgic theme based on everyone's favourite encyclopedia.]

The [[Wikipedia]] theme for Obsidian was created on December 16, 2022. The theme was initially created as a CSS test, but ended up being developed into a fully-fledged theme that was quickly published to the [[Community Themes]]. 



The theme took one week to complete initially, and a great amount of the CSS styles were taken directly from Wikipedia. The theme is licensed under the [[MIT License]] and now supports both Light and Dark modes.

# Supported Plug-ins
The theme supports and contains unique styles for the following:
1. [[Dataview]]
2. [[Style Settings]]
3. [[Page Gallery]]
4. [[Cardboard]]
5. [[Folder Note]]


# Notes
- A good portion of the CSS was actually taken directly from [[Wikipedia]] and mapped to components in Obsidian.
- The theme, in a way, turns **off** readable line length by default.
```

On some devices, images might not float correctly unless the Style Settings plugin is installed.

**Comment by vec715** [comment]
![Image](https://github.com/user-attachments/assets/fe45d598-298f-4754-95b6-27f4e2a9307d)

Thank you so much! I ran into the same issue when I tried doing this manually a few days ago. the right panel takes up the entire space, so the text can’t be displayed alongside it

**Comment by Bluemoondragon07** [comment]
@vec715 You must be in edit mode.  It only floats beside the text fully in Reading View because it would otherwise interfere with cursor movement when editing text.

**Comment by TheLilaDoez** [comment]
Hey just wanted to check if there was an example (or even possibility) to list off details off a category? Like how in the 'Supported Themes' there is basically a list of 'Light' and then 'Dark', how would you put them as a vertical list? And would there be a way to nest them as sub bullet points? I did get to do the first bullet point but how to start of the vertical list is what is getting me stuck ;-;

**Comment by Chartreuse310** [comment]
not sure what I've missed but here's what it looks like in my vault (with no plugin or other css setting) 😢
<img width="1563" height="1019" alt="Image" src="https://github.com/user-attachments/assets/4a1bf629-bc4e-4bf5-a978-ef6d25ce4633" />

**Comment by ninjakai03** [comment]
> not sure what I've missed but here's what it looks like in my vault (with no plugin or other css setting) 😢

i was JUST about to comment this, seems to be broken...

**Comment by Bluemoondragon07** [comment]
@Chartreuse310 @ninjakai03  The [key::data] format is for the dataview plugin, so there is no selector for the theme to style for that syntax unless dataview is installed. To get the same look without dataview, you can put a H6 inside the callout for the yellow-background lines, like:

```markdown
>[!info]
>###### key
>data
```

**Comment by ninjakai03** [comment]
OHHHHHHHHHHH OKAY thank you so much!!!

**Comment by enderfare** [comment]
How do you centerr the items inseide of a block

**Comment by mcv1cky** [comment]
Hi! Thanks for the theme, it's wonderful. I thought I'd ask here since it's related to the example document. How do I omit the "ⓘ Info" part at the top of the infobox to get the same look as in the readme's first image and actual wikipedia pages? I'm getting this

<img width="326" height="384" alt="Image" src="https://github.com/user-attachments/assets/05e43bcd-0ab2-424d-a659-c44301aa354d" />

from this

```
>[!info]
> ![[newest.png]]
> An image
>[category::a category]
```

**Comment by Bluemoondragon07** [comment]
@victoriaemc If you have style settings installed, under "Typography", you can choose to hide the info icon in the callouts
 ![Screenshot_20251102-170208_Obsidian.png](https://github.com/user-attachments/assets/04a628b9-8be3-4a64-be7d-ece5dedc307a)
